manager/controllers/common_test.go

```diff
@@ -36,11 +37,16 @@ func CreateRequest(method string, url string, body io.Reader, contentType string
 }
 
 // Create request and initialize router with params
-func CreateRequestRouterWithParams(method string, url string, body io.Reader, contentType string,
-	handler gin.HandlerFunc, routerAccount int, routerMethod string, routerPath string,
+func CreateRequestRouterWithParams(method string, routerPath string, param string, queryString string,
+	body io.Reader, contentType string, handler gin.HandlerFunc, routerAccount int,
 	contextKVs ...core.ContextKV) (w *httptest.ResponseRecorder) {
+	if len(queryString) > 0 && queryString[0] == '/' {
+		queryString = queryString[1:]
+	}
```

Comment:
I think this `if` actually workarounds wrong input data. Query string should not start with '/' so it would be better to fix input data and remove the `if`.
I see there are only 2 occurrences of  "/?offset=1000" which should be "?offset=1000"
advisories_test.go:302
packages_test.go:86